user-manual/import-export/upload-digital-object.rst

```diff
@@ -191,6 +193,186 @@ your :term:`digital object`.
 
 :ref:`Back to top <upload-digital-object>`
 
+
```

Comment:
Hmmm, isn't this content already in a separate PR? I think there might have been some issues with how you tried to separate out your commits... might need to do some git interactive rebase fuckery to resolve that - Steve has helped me out in the past when I've run into this.

Reply:
Ah, this is how the two branches got wrapped up into this one PR. I guess I had noticed I was missing a "Back to top" link so added it while I had already moved over to the Download AIP branch. Git recognized that the file was part of the the previous commit/PR and bundled it. Yes, we'll need some help to untangle.